src/Symfony/Component/AssetMapper/ImportMap/Resolver/JsDelivrEsmResolver.php

```diff
@@ -26,7 +26,7 @@ final class JsDelivrEsmResolver implements PackageResolverInterface
     public const URL_PATTERN_DIST = self::URL_PATTERN_DIST_CSS.'/+esm';
     public const URL_PATTERN_ENTRYPOINT = 'https://data.jsdelivr.com/v1/packages/npm/%s@%s/entrypoints';
 
-    public const IMPORT_REGEX = '{from"/npm/((?:@[^/]+/)?[^@]+?)(?:@([^/]+))?((?:/[^/]+)*?)/\+esm"}';
+    public const IMPORT_REGEX = '#(?:import|export)\s*(?:\{[^}]*\}|\w+)?\s*\**[from]*("/npm/((?:@[^/]+/)?[^@]+?)(?:@([^/]+))?((?:/[^/]+)*?)/\+esm")#';
```

Comment:
`[from]*` is wrong as this would match `f`, `morf` or `ffffffffffffffffffffffffffffffffffffffffffffff`. Having an optional `from` keyword is done with `(?:from)?` (or `(from)?` but this captures it).

Reply:
resolved thanks to your suggestion below